lib/_http_outgoing.js

```diff
@@ -552,7 +552,7 @@ function processHeader(self, state, key, value, validate) {
   if (ArrayIsArray(value)) {
     if (
       (value.length < 2 || !isCookieField(key)) &&
-      (!self[kUniqueHeaders] || !self[kUniqueHeaders].has(key.toLowerCase()))
+      (!self[kUniqueHeaders] || !self[kUniqueHeaders].has(getHeaderFieldLowercase(key)))
```

Comment:
By the benchmark, I think we should revert this change, I'm not sure (since I'm reading on mobile) but this should be the cause of the slowdown on benchmark of headers